src/Welcome.spec.tsx

```diff
@@ -20,13 +20,13 @@ jest.mock("react-native-webview", () => {
     },
   }
 })
-jest.mock("../locales/eula/en.html", () => {
```

Comment:
Do we still need these? Maybe we can run a quick cleanup after since we are only using the links on that terms of service screen right?

Reply:
It appears we don't! Nice catch!